README.md

```diff
@@ -1,43 +1,45 @@
-# RockLib.Immutable [![Build status](https://ci.appveyor.com/api/projects/status/e217hl1l8yaj7du7?svg=true)](https://ci.appveyor.com/project/RockLib/rocklib-immutable) [![NuGet](https://img.shields.io/nuget/vpre/RockLib.Immutable.svg)](https://www.nuget.org/packages/RockLib.Immutable)
+# RockLib.Immutable
 
-This library contains the `Semimutable` class, which represents a "semimutable" value. Its value can be changed either via the setter of the <see cref="Value"/> property, or by calling the <see cref="SetValue"/> method. However, once the getter of the <see cref="Value"/> property is accessed or the <see cref="LockValue"/> method is called, the value will never change again.
+This library contains the `Semimutable` class, which represents a "semimutable" value. It's value can change either via the setter of the <see cref="Value"/> property, or by calling the <see cref="SetValue"/> method. Once the getter of the <see cref="Value"/> property is accessed or the <see cref="LockValue"/> method is called, the value will never change again.
```

Comment:
'can change' seems like a different meaning.

Maybe just It's value can be set either...?

Reply:
yeah I  see what your saying. let me take another shot at it.

Reply:
updated